README.md

```diff
@@ -9,24 +9,61 @@ sequencing data such as [SAM][2] and [VCF][3].
 
 As of version 2.0.1 (Nov. 2015) Picard requires Java 1.8 (jdk8u66). The last version to support Java 1.7 was release 1.141.
 
-To clone and build:
-Clone the repo:
+####Building Picard
 
+* First, clone the repo:
+```
     git clone git@github.com:broadinstitute/picard.git
     cd picard/
+```
+
+* Picard is now built using [gradle](http://gradle.org/). A wrapper script (`gradlew`) is included which will download the appropriate version of gradle on the first invocation.
+    
+* To build a fully-packaged, runnable Picard jar with all dependencies included, run:
+```
+    ./gradlew shadowJar
+```
+
+* The resulting jar will be in `build/libs`. To run it, the command is:
+```
+    java -jar build/libs/picard-public*.jar
```

Comment:
oh, bleh, we don't have a project name set in settings.gradle so the jar name depends on whatever folder you're in... we should fix that...